### DISCUSSION TITLE
Apply .editorconfig when running nx commands

### DISCUSSION THREAD

**Discussion by LuckeeDev** [nrwl/nx#10119] [General]
Whenever I run an NX command, it messes up the formatting of all my files because it doesn't take into account the settings of my .editorconfig file. For example: I have set the workspace to use tabs, but running an nx command formats my package.json with two spaces instead of tabs. Is there any way to fix this?

**Comment by marko-hologram** [comment]
Did you maybe try adding a [Prettier](https://prettier.io/) config? I know it might be redundant to define another config file for some formatting, but it maybe is a temporary workaround for this. Define a Prettier config that has tabs set to true and NX will potentially respect that config instead.

**Comment by LuckeeDev** [comment]
@marko-hologram I tried, but unfortunately NX doesn't respect Prettier config. This is my `.prettierrc` file:
```
{
	"singleQuote": true,
	"useTabs": true
}
```

**Reply by marko-hologram** [reply]
Interesting. The way it's presented in the docs, I would expect it to use user defined Prettier config https://nx.dev/migration/manual#formatting. Maybe it only picks it up when running formatting command manually?
